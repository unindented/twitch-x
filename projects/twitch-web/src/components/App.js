import React from 'react'
import PropTypes from 'prop-types'
import Root from 'twitch-ui/src/components/Root'
import Navigation from 'twitch-ui/src/components/Navigation'
import Main from 'twitch-ui/src/components/Main'

export default function App (props) {
  const {children} = props

  return (
    <Root>
      <Navigation />
      <Main>
        {children}
      </Main>
    </Root>
  )
}

App.propTypes = {
  children: PropTypes.node
}
