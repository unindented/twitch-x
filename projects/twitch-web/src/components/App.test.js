import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import App from './App'

describe('App', () => {
  let tree

  beforeEach(() => {
    const renderer = new ShallowRenderer()
    renderer.render(<App />)
    tree = renderer.getRenderOutput()
  })

  it('renders', () => {
    expect(tree).toMatchSnapshot()
  })
})
