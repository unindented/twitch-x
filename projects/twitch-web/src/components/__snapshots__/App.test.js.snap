// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<styled.div>
  <Navigation />
  <styled.main>
    foo
  </styled.main>
</styled.div>
`;
